bin/Cargo.toml

```diff
@@ -60,6 +60,9 @@ tls = ["dns-over-openssl"]
 # WARNING: there is a bug in the mutual tls auth code at the moment see issue #100
 # mtls = ["trust-dns-client/mtls"]
 
+webpki-roots = ["trust-dns-client/webpki-roots"]
+native-certs = ["trust-dns-client/native-certs"]
```

Comment:
I'm not sure what we want here, should we enable one of them by default?
Or just use root certificates without requiring any crate feature at all?

Reply:
I think adding `native-certs` to default probably makes sense.

Reply:
Done.